CHANGELOG.md

```diff
@@ -29,6 +29,16 @@ instructions](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds-clus
   - Update `cumulus-rds-tf` module to take additional parameters in support of
     migration from Aurora PostgreSQl v11 to v13.   See Migration Notes for more details.
 
+## Changed
+
+## [v16.1.4] 2024-2-16
+
+Please note changes in 16.1.4 may not yet be released in future versions, as this
+is a backport/patch release on the 16.x series of releases. Updates that are
+included in the future will have a corresponding CHANGELOG entry in future releases.
+
+### Changed 
```

Comment:
You had the correct ticket here before. Need that entry for CUMULUS-3547 under 16.1.4 and not under "unreleased".

Reply:
Remove this subheading.

Reply:
```suggestion

```